src/main/java/edu/harvard/iq/dataverse/api/Search.java

```diff
@@ -136,7 +136,7 @@ public Response search(
                         paginationStart,
                         dataRelatedToMe,
                         numResultsPerPage,
-                        queryEntities
+                        true //SEK get query entities always for search API additional Dataset Information 6300  12/6/2019
```

Comment:
If "true" vs. "false" for `query_entities` has no affect in this pull request, the documentation should be changed to reflect this. Here's a screenshot of how it's documented as of 4.18.1:

![Screen Shot 2019-12-06 at 3 05 25 PM](https://user-images.githubusercontent.com/21006/70352906-2ff55b00-183a-11ea-9d02-941008b65cd7.png)

Reply:
I updated the parameter list (removed the reference to query_entities) and added an example with the new data items shown.